### ISSUE TITLE
Use MPI with create_observations

### ISSUE THREAD

**Issue by paganol** [litebird/litebird_sim#109]
Hi there, I'm trying to use mpi with sim.create_observations. I cannot understand how to pass correctly the communicator. Setting it in the initialization of lbs.Simulation is not working. Am I supposed to initialize obs only with lbs.Observation for handling mpi runs? It should not be the case, right?
Another small thing, reading the documentation, lbs.Simulation should have the flag use_mpi, but it doesn't exist anymore.

**Comment by dpole** [comment]
Hi Luca, yes, everything should be set for you when you construct the simulation with an MPI communicator. Note that you have to set ``distribute=False``. I'd remove this flag altogether, I find it misleading.
About the doc, I believe you are right and it should be fixed. @ziotom78 do you agree?

**Comment by ziotom78** [comment]
@dpole, yes, I agree both with the removal of `distribute` and of `use_mpi`. I'll do it this morning

**Comment by paganol** [comment]
Ok, thank you both
